js/test/test.js

```diff
@@ -227,6 +227,7 @@ let testExchange = async exchange => {
 
     await tests['fetchFundingFees']  (exchange)
     await tests['fetchTradingFees']  (exchange)
+    await tests['fetchStatus'] (exchange)
```

Comment:
After thinking more about it, we should not really try testing the exchange, if `fetchStatus` is true and the `status` is not `ok` ) But I'll merge it this way for now, and will rework the test a bit later.